packages/angular-ivy/package.json

```diff
@@ -22,9 +22,10 @@
   },
   "dependencies": {
     "@sentry/browser": "7.53.1",
+    "@sentry/core": "7.53.1",
```

Comment:
l/q: why do we need core now explicitly?

Reply:
got some type errors during build here - I'll try again if it is actually still needed or not anymore (had to tweak around a bit to get this working)

Reply:
OK, I tried again and actually it is not needed, removing it again!